src/Mod/Mesh/Gui/Resources/icons/Mesh_Fill_up_Holes.svg

```diff
@@ -0,0 +1,220 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:xlink="http://www.w3.org/1999/xlink"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   version="1.1"
+   inkscape:output_extension="org.inkscape.output.svg.inkscape"
+   sodipodi:docname="Mesh_Fill_Holes.svg"
+   inkscape:version="1.0 (4035a4fb49, 2020-05-01)"
+   sodipodi:version="0.32"
+   id="svg9484"
+   height="64"
+   width="64">
```

Comment:
I think we're supposed to used "plain" svg rather Inkscape specific SVG.
Please do SaveAs... and choose "PlainSVG" type.

Reply:
https://www.freecadweb.org/wiki/Artwork_Guidelines should include this requirement...

Reply:
@bitacovir Thanks, added: https://wiki.freecadweb.org/Artwork_Guidelines#Recommended_recording_format
@vocx-fc FYI